src/api/clearlyDefined.js

```diff
@@ -133,7 +124,7 @@ function post(url, token, payload) {
   return fetch(url, {
     headers: getHeaders(token),
     method: 'POST',
-    body: JSON.stringify(payload)
+    body: JSON.stringify(payload),
```

Comment:
is this a linting thing to _add_ trailing ','?

Reply:
Habit. Did we decide on a formatter for this repo? I don't see one. `prettier`?

Reply:
+1 for `prettier`!

Reply:
Agreed. I'll quick-fix these format issues you've noted in this review, but let's save the format pass for another commit.